tools/MatrixHelper/MetricData.cs

```diff
@@ -18,16 +18,21 @@ namespace MatrixHelper;
 
 internal class MetricData
 {
-    public MetricData(string metricName, string instrument, string description)
+    private const string Custom = "Custom metric.";
```

Comment:
Wording needs to be confirmed with @aurbiztondo-splunk

Reply:
LGTM